sdk/core/azure-core/test/libcurl-stress-test/libcurl_stress_test.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Validates the Azure Core transport adapters with fault responses from server.
+ *
+ * @note This test requires the Http-fault-injector
+ * (https://github.com/Azure/azure-sdk-tools/tree/main/tools/http-fault-injector) running. Follow
+ * the instructions to install and run the server before running this test.
+ *
+ */
+#define REQUESTS 100
+#define WARMUP 100
+#define ROUNDS 100
+
+#include <azure/core.hpp>
+#include <azure/core/http/curl_transport.hpp>
```

Comment:
I would have expected you'd need to include this header:
https://github.com/Azure/azure-sdk-for-cpp/blob/e0706ba1fba166fbd85de980ded84648a468e338/sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp#L133